swagger/api.yaml

```diff
@@ -66,13 +66,13 @@ paths:
   # Enterprise Access IDA
   # These are served from the enterprise-access IDA, but we surface them with the rest of the enterprise endpoints
   "/enterprise/v1/assignment-configurations/{assignment_configuration_uuid}/admin/assignments/cancel/":
-    $ref: "https://raw.githubusercontent.com/openedx/enterprise-access/8975f08/api.yaml#/endpoints/v1/learnerContentAssignmentCancelRequest"
+    $ref: "https://raw.githubusercontent.com/openedx/enterprise-access/7b79f45/api.yaml#/endpoints/v1/learnerContentAssignmentCancelRequest"
```

Comment:
Any reason of updating hash in the cancel API's?

Reply:
This is just to keep all enterprise-access endpoints point the same version to avoid any unexpected test failures. This happened with me once so being cautious. :)